Getting-Started/Backoffice/Variants/index.md

```diff
@@ -0,0 +1,36 @@
+---
+versionFrom: 8.0.0
+---
+
+# Variants
+
+Variants is a new feature included in Umbraco v8. It allows you to vary content by culture, so you can allow a content node to exist in several languages.
+
+## How to enable variants
+
+The first thing to ensure when you want to work with variants is that you have more than one language enabled, this can be done from the `settings` section:
```

Comment:
```suggestion
The first thing to ensure when you want to work with variants is that you have more than one language enabled, this can be done from the `Settings` section:
```